src/inference.jl

```diff
@@ -1986,3 +1408,397 @@ function rxinference(;
 
     return engine
 end
+
+function rxinference(; kwargs)
+    @warn "The `rxinference` function is deprecated and will be removed in the future.  Use `infer` with autoupdates keyword argument instead."
```

Comment:
```suggestion
    @warn "The `rxinference` function is deprecated and will be removed in the future.  Use `infer` with the `autoupdates` keyword argument instead."
```

Reply:
I suggest we create an associated issue regarding the documentation page after merging this PR.